contrib/metadata-ingestion/python/mysql-etl/README.md

```diff
@@ -0,0 +1,50 @@
+# MCE Producer/Consumer CLI
```

Comment:
See https://github.com/linkedin/datahub/pull/1805 for context. TL;DR is we're moving py-based ETL to contrib one bit at a time.

Reply:
I see, thanks for the context. Sounds good. Ideally, it would even be better to have a separate repo of metadata-ingestion-examples. But, probably we can't create so many repos under linkedin org. Maybe, datahub org is a way to go long term :smile: